### PR TITLE
 hacker rank problem 01

### PR DESCRIPTION
**Pull request by rukshand2001**

### CODE REVIEW

File: src/main/java/org/example/JavaConditions.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+            System.out.println("Weird");
+        } else if (enterNumber >= 2 && 5>= enterNumber){
+            System.out.println("Not Weird");
+        }else if(enterNumber>=6 && enterNumber<=20){
+            System.out.println("Weird");
+        } else if (enterNumber>20) {
+            System.out.println("Not Weird");
+        }
+    }
+}
```
Comment (aselarbd) [inline src/main/java/org/example/JavaConditions.java:48]: add new line EOF

File: src/main/java/org/example/JavaConditions.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        if(enterNumber%2 == 1 ){
+            System.out.println("Weird");
+        } else if (enterNumber >= 2 && 5>= enterNumber){
+            System.out.println("Not Weird");
+        }else if(enterNumber>=6 && enterNumber<=20){
+            System.out.println("Weird");
+        } else if (enterNumber>20) {
+            System.out.println("Not Weird");
+        }
+    }
```
Comment (aselarbd) [inline src/main/java/org/example/JavaConditions.java:46]: close the scanner

File: src/main/java/org/example/JavaConditions.java [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+import java.util.Scanner;
+
+public class JavaConditions {
+
+    public static void main(String[] args) {
+
+        Scanner scanner = new Scanner(System.in);
+
+        System.out.print("Enter number: ");
+        int enterNumber = scanner.nextInt();
```
Comment (aselarbd) [inline src/main/java/org/example/JavaConditions.java]:
```suggestion
        int enteredNumber = scanner.nextInt();
```

File: src/main/java/org/example/JavaConditions.java [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+import java.util.Scanner;
+
+public class JavaConditions {
+
+    public static void main(String[] args) {
+
+        Scanner scanner = new Scanner(System.in);
+
+        System.out.print("Enter number: ");
```
Comment (aselarbd) [inline src/main/java/org/example/JavaConditions.java]:
```suggestion
        System.out.print("Enter a number: ");
```

File: src/main/java/org/example/JavaConditions.java (hunk 1 above)
Comment (rukshand2001) [reply src/main/java/org/example/JavaConditions.java:48]: done

File: src/main/java/org/example/JavaConditions.java (hunk 2 above)
Comment (rukshand2001) [reply src/main/java/org/example/JavaConditions.java:46]: done

File: src/main/java/org/example/JavaConditions.java (hunk 3 above)
Comment (rukshand2001) [reply src/main/java/org/example/JavaConditions.java]: done

File: src/main/java/org/example/JavaConditions.java (hunk 4 above)
Comment (rukshand2001) [reply src/main/java/org/example/JavaConditions.java]: done

**Review by aselarbd — APPROVED** [review]

**Pull request merged**